datasets/dev_org/test_data.sql

```diff
@@ -687,160 +689,174 @@ CREATE TABLE "Contact" (
+INSERT INTO "General_Accounting_Unit__c" VALUES(18,'The ut at Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(19,'The et Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(20,'The ultrices Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(21,'The pellentesque viverra Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(22,'The adipiscing elit Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(23,'The donec dapibus Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(24,'The hac habitasse Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(25,'The sit Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(26,'The non Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(27,'The id sapien Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(28,'The purus phasellus Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(29,'The in Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(30,'The in Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(31,'The elit Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(32,'The pretium nisl Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(33,'The a Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(34,'The integer aliquet Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(35,'The magna Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(36,'The consequat Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(37,'The dapibus Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(38,'The ipsum Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(39,'The sit Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(40,'The et Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(41,'The nulla Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(42,'The nullam Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(43,'The proin eu Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(44,'The erat Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(45,'The eget Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(46,'The eleifend pede Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(47,'The felis eu Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(48,'The tempus sit Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(49,'The lobortis sapien Fund');
+INSERT INTO "General_Accounting_Unit__c" VALUES(50,'The augue quam Fund');
+CREATE TABLE "Level__c" (
+	id INTEGER NOT NULL, 
+	"Name" VARCHAR(255), 
+	"Active__c" VARCHAR(255), 
+	"Description__c" VARCHAR(255), 
+	"Level_Field__c" VARCHAR(255), 
+	"Maximum_Amount__c" VARCHAR(255), 
+	"Minimum_Amount__c" VARCHAR(255), 
+	"Previous_Level_Field__c" VARCHAR(255), 
+	"Source_Field__c" VARCHAR(255), 
+	"Target__c" VARCHAR(255), 
+	PRIMARY KEY (id)
+);
+INSERT INTO "Level__c" VALUES(1,'Gold Donor','true','','Current_Level__c','','10000.0','Previous_Level__c','npo02__TotalOppAmount__c','Contact');
```

Comment:
We aren't mapping the field `Current_Level__c`, and I'm not sure where it's defined (same for `Previous_Level__c`). Is this a concern?

Reply:
I don't think so. Those are unmanaged fields that get deployed into the org anyway. I think of these like Rollup fields, which we don't map either. In terms of performance, what would be really cool (and would also address this), is to disable rollups before inserting/updating Opps, and then run the rollup and levels job after the data load has completed. I think that's possible with some apex-anon that runs as part of the task and/or flow.

Reply:
It looks like this field is defined in `config/qa`, which means this dataset comes with a dependency on that unpackaged metadata set, and that set is not currently deployed in `dev_org`. Am I missing another source for it?

Reply:
Ah. That is a really good point. The data load won't fail if the fields don't exist because these are just text fields with those field names, however the Levels job would crash when run if the fields aren't there. 

Darn. I was hoping to include a Level in the data to provide more realistic test data. I will remove the Level__c object from the mapping/sql.